WordPress/JetpackStatsWidgetsCoreTests/WidgetUrlSourceTests.swift

```diff
@@ -13,16 +14,4 @@ final class WidgetUrlSourceTests: XCTestCase {
         let widgetUrl = url.appendingSource(.lockScreenWidget)
         XCTAssertEqual(widgetUrl.absoluteString, "https://test?source=lockscreen_widget")
     }
-
-    func testHomeScreenWidgetSourceType() {
-        let source = WidgetUrlSource.homeScreenWidget.rawValue
-        let deepLinkSource = DeepLinkSource(sourceName: source)
-        XCTAssertEqual(deepLinkSource, .widget)
-    }
-
-    func testLockScreenWidgetSourceType() {
-        let source = WidgetUrlSource.lockScreenWidget.rawValue
-        let deepLinkSource = DeepLinkSource(sourceName: source)
-        XCTAssertEqual(deepLinkSource, .lockScreenWidget)
-    }
```

Comment:
`DeepLinkSource` belongs in the WordPress target, at least for the moment until we might move it to a dedicated underlying one.

By testing its interop with `WidgetUrlSource` at the WordPressTests level we guarantee we still have that logic checked without having to jump through hoops to get `DeepLinkSource` in the widgets target.